spec/customer_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'customer'
+require 'menu'
+
+class Customer_Class; include Customer; end
+
+describe Customer do
+  let(:customer){Customer_Class.new}
+
+it 'should receive a Twilio text with delivery time' do
+  customer.send_text
+  #check phone to confirm that a customer can receive text
+end
+
```

Comment:
You should be using a stub, so that you are not spamming the Twilio API every time you run your tests.